Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -89,9 +89,32 @@ @interface AVSampleBufferDisplayLayer (Staging_100128644)
     , m_logIdentifier(player->mediaPlayerLogIdentifier())
     , m_videoLayerManager(makeUnique<VideoLayerManagerObjC>(m_logger, m_logIdentifier))
     , m_listener(WebAVSampleBufferListener::create(*this))
+    , m_seekTimer(*this, &MediaPlayerPrivateWebM::seekInternal)
 {
     ALWAYS_LOG(LOGIDENTIFIER);
     m_parser->setLogger(m_logger, m_logIdentifier);
+
+    // addPeriodicTimeObserverForInterval: throws an exception if you pass a non-numeric CMTime, so just use
+    // an arbitrarily large time value of once an hour:
```

Comment:
Why is an hour large enough?
Does it mean we might have an issue every one hour or so?
I would tend to put something bigger then, maybe 24 hours.

Reply:
As above, this is replicating the code from `MediaPlayerPrivateMediaSourceAVFObjC`

if this is to be removed, I will do so as a follow-up and remove it wherever it's left